cypress/support/step-definitions/accessibility/a11y-utils.js

```diff
@@ -0,0 +1,112 @@
+import {
+  commonButtonPreviewRoot,
+  getDataElementByValue,
+} from "../../../locators";
+import { accordionDefaultTitle } from "../../../locators/accordion";
+import {
+  actionPopoverButton,
+  actionPopoverWrapper,
+} from "../../../locators/action-popover";
+import { popoverSettingsIcon } from "../../../locators/popover-container";
+import { visitComponentUrl } from "../../helper";
+import storiesJSON from "../../../../storybook-static/stories.json";
+
+const A11YOptions = {
+  runOnly: {
+    type: "tag",
+    values: [
+      // "wcag2a", // WCAG 2.0 & WCAG 2.1 Level A
```

Comment:
These should all be enabled.

Reply:
nope, we will enable it after merge - because will have a lot of failures